centos-IPA-ssh/centos-IPA-ssh.md

```diff
@@ -0,0 +1,58 @@
+# Enable SSH to centos IPA node using dynamic-login feature
+
+## Investigation status 12.10.2021
```

Comment:
Investigation status 12.10.2021 - > Goal of the documentation
I think there is no need to write date as the document is version controlled.